includes/shortcodes.php

```diff
@@ -631,7 +632,14 @@ function give_totals_shortcode( $atts ) {
 			$form_ids = array_filter( array_map( 'trim', explode( ',', $atts['ids'] ) ) );
 		}
 
-        array_walk($form_ids, '_give_redirect_form_id');
+        array_map(
```

Comment:
When using `array_map`, you must assign the result to a variable. Otherwise, it will have no effect. In this case, please assign it to the variable `$form_ids`.